lib/hardware_services/network_drivers/hilapi.py

```diff
@@ -0,0 +1,107 @@
+# this is a library to conveniently do REST calls to HIL server
+
+import requests
+import sys
+import yaml
+
+def error_check(response):
```

Comment:
It will take no time to put this into a class. I don't have any preferences. Do you want me to make a class with only static methods?